serverless.yml

```diff
@@ -0,0 +1,41 @@
+service: sls-provider
+
+frameworkVersion: ">=1.1.0 <2.0.0"
+
+provider:
+  name: aws
+  runtime: nodejs10.x
+
+package:
+  include:
+    # Root level
+    - "!*"
+    - "package.json"
+    # Source directories
+    - "!aws/**"
+    - "!terraform"
```

Comment:
GLOBAL: I updated the issue/docs in jetpack, but, but you don't need:

```yaml
- "!terraform"
- "!terraform/**"
```

You only need:

```yaml
- "!terraform/**"
```